packages/template/webpack-typescript/package.json

```diff
@@ -13,7 +13,9 @@
   "dependencies": {
     "@electron-forge/shared-types": "7.9.0",
     "@electron-forge/template-base": "7.9.0",
-    "fs-extra": "^10.0.0"
+    "fs-extra": "^10.0.0",
+    "typescript": "~5.4.5",
+    "webpack": "^5.69.1"
```

Comment:
How come we needed to add these to the template?

Reply:
Adding these lines removes the yarn warning
```
➤ YN0002: │ @electron-forge/template-webpack-typescript@workspace:packages/template/webpack-typescript doesn't provide typescript (pd370b3), requested by fork-ts-checker-webpack-plugin.
➤ YN0002: │ @electron-forge/template-webpack-typescript@workspace:packages/template/webpack-typescript doesn't provide webpack (p738c54), requested by fork-ts-checker-webpack-plugin.
```

I'll remove them to avoid additional explicit dependencies, but IMO they should probably be added into the `devDependencies` so it doesn't just work via workspace magic

Reply:
Ah, gotcha.

I think since `fork-ts-checker-webpack-plugin` is listed as a `devDep`, it makes sense to have its requested peer deps as `devDeps` as well. 👍